core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -207,6 +209,7 @@ registry.ci.openshift.org/origin/4.7:openstack-cloud-controller-manager quay.io/
 registry.ci.openshift.org/origin/4.7:openstack-installer quay.io/openshift/origin-openstack-installer:4.7 quay.io/openshift/origin-openstack-installer:4.7.0
 registry.ci.openshift.org/origin/4.7:openstack-machine-controllers quay.io/openshift/origin-openstack-machine-controllers:4.7 quay.io/openshift/origin-openstack-machine-controllers:4.7.0
 registry.ci.openshift.org/origin/4.7:operator-lifecycle-manager quay.io/openshift/origin-operator-lifecycle-manager:4.7 quay.io/openshift/origin-operator-lifecycle-manager:4.7.0
+registry.ci.openshift.org/origin/4.7:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.7 quay.io/openshift/origin-operator-marketplace:4.7.0
```

Comment:
Hrm, these should be excluded, so I think the script is broken.  It may be that the "okd" lookup in the script that filters them is still pointing at the wrong namespace